lib/flipper/ui/action_collection.rb

```diff
@@ -1,20 +0,0 @@
-module Flipper
```

Comment:
sounds great, going to make api/action_collection again, and put this back under ui::